pkgs/by-name/na/navidrome/plugins/example-builder.nix

```diff
@@ -0,0 +1,27 @@
+{
+  name,
+}:
+{
+  buildGo124Module,
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  navidrome,
+}:
```

Comment:
These can probably be joined into a single set of args, just move the `inherit name` to the outer attrset.